interp/interp.go

```diff
@@ -751,8 +758,8 @@ func ignoreScannerError(e *scanner.Error, s string) bool {
 // rename packages.
 func (interp *Interpreter) ImportUsed() {
 	sc := interp.universe
-	for k := range interp.binPkg {
-		name := key2name(k)
+	for k, pkg := range interp.binPkg {
+		name := key2name(pkg[".name"].String())
```

Comment:
Is it a leftover from the previous commit before we decided to store the package name in the key, instead of in .name?
If not, I do not understand what that is for.

Reply:
Yes, a leftover from previous attempt. It should be removed.